app/components/WeatherDataComponent/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * WeatherDataComponent
+ *
+ */
+
+import React, { memo } from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
```suggestion
// import PropTypes from 'prop-types';
// import styled from 'styled-components';
```

remove unused code, don't comment them out